packages/components/src/theme/ComposerTheme/Components/Button.ts

```diff
@@ -0,0 +1,98 @@
+import { ComponentStyleConfig } from '@chakra-ui/theme';
+
+export const Button: ComponentStyleConfig = {
+  baseStyle: {
+    fontWeight: 'medium',
+    borderRadius: 'base', // <-- border radius is same for all variants and sizes
```

Comment:
this has to use the corresponding radii defined in our theme

Reply:
as we are not defined a specific radius I am using the default one.  at the Chakra documentation (https://chakra-ui.com/docs/theming/theme#border-radius), the md value is the one that follows the Figma definitions

Reply:
we do have specific radius